pageflow.gemspec

```diff
@@ -118,6 +120,9 @@ Gem::Specification.new do |s|
   # Used by the dummy rails application
   s.add_development_dependency 'mysql2', '~> 0.3.16'
 
+  s.add_development_dependency 'bundler'
+  s.add_development_dependency 'rake'
+
```

Comment:
The rest of the gemspec goes with pessimistic version constraints. Should we state that we don't know yet whether we are going to be compatible with Bunder 2 or Rake 13?

Reply:
I wasn't sure. I am not aware of their roadmaps and don't want to get into it either.

Restricting these pessimistically felt like busywork in the future, but I guess consistency wins here. For what it's worth, Rails 4 needed bundler < 2 but Rails 5 wants 1.3.0 and higher.

Maybe we'll roll with `rake ~> 11.x` and `bundler >= 1.3.0`?

Reply:
That would exclude the current Rake version 12. Since there are no known (to me) Rake or Bundler versions that do not work with Pageflow (not the Rails version used by Pageflow) `rake < 13` and `bundler < 2` would probably be the most consistent pessimistic choices. But I'm also open to using open ranges given that both are development dependencies and core tools that should be pretty stable going forward.

Reply:
I've been wrestling with this and I came to think that I should just remove them from `pageflow.gemspec`. Rails is a dependency and Rails will do a fine job of pulling in Rake and Bundler.

Reply:
Fine with me. We also use a lot of other stuff - even at runtime, not just for development - that comes with Rails (e.g. ActiveRecord) without explicitly declaring dependencies.